build.sbt

```diff
@@ -71,7 +75,6 @@ def coreProject(includePlayVersion: String): Project = {
   val path = s"play$playSuffix-core"
   commonProject(path, path).settings(
     name := s"play$playSuffix-test-ops-core",
-    scalaVersion := scalaVersions.head,
```

Comment:
I'm not certain this was broken to begin with, now that I double check it:
https://bintray.com/rallyhealth/maven/play27-test-ops-core/1.1.3#files/com%2Frallyhealth%2Fplay27-test-ops-core_2.13%2F1.1.3